onnx/backend/test/case/node/mish.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Mish(Base):
+
+    @staticmethod
+    def export() -> None:
+        node = onnx.helper.make_node(
+            'Mish',
+            inputs=['X'],
+            outputs=['Y']
+        )
+
+        input_data = np.array([[[[0.8439683], [0.5665144], [0.05836735]],
+            [[0.02916367], [0.12964272], [0.5060197]],
+            [[0.79538304], [0.9411346], [0.9546573]]],
+            [[[0.17730942], [0.46192095], [0.26480448]],
+            [[0.6746842], [0.01665257], [0.62473077]],
+            [[0.9240844], [0.9722341], [0.11965699]]],
+            [[[0.41356155], [0.9129373], [0.59330076]],
+            [[0.81929934], [0.7862604], [0.11799799]],
+            [[0.69248444], [0.54119414], [0.07513223]]]], dtype=np.float32)
```

Comment:
Some negative values could be included in the test, just in case. It's up to you, I don't think it's necessary.